File: python/rabbitmq/publisher.py
from datetime import datetime
from modules.rabbitmq.connector import RabbitMQ

amqp = RabbitMQ()
connection = amqp.connect('localhost', 5672, 'guest', 'guest')
channel =  amqp.create_channel(connection)

queue='some-queue'
headers= {
  "timestamp": str(datetime.timestamp(datetime.now()))
}

payload = "{'name': 'John Doe', 'age': '26'}"

amqp.publish(channel,queue, payload, headers)

connection.close()

File: ruby/rabbitmq/publisher.rb
require 'bunny'

connection = Bunny.new(:host => "localhost", :vhost => "/", :user => "guest", :password => "guest")

def send(channel, queue, message, headers)
  queue = channel.queue(queue)
  exchange = channel.fanout('logging.events')
  queue.bind(exchange)
  queue.publish(
    message,
    :headers  => headers,
    :delivery_mode =>  1
  )
  puts " [x] Sent #{message}"
end

connection.start

queueName = 'to-be-processed'
headers = {
  :timestamp  => Time.now.to_i
}
payload = "Message published with Ruby!"
channel = connection.create_channel
send(channel, queueName, payload, headers)

connection.close